core/web/Composition.php

```diff
@@ -246,7 +246,13 @@ public function getPrefixPath()
      */
     public function createRouteEnsure(array $overrideKeys = [])
     {
-        return $this->hidden || (!$this->hidden && $this->langShortCode == $this->defaultLangShortCode && $this->hideDefaultPrefixOnly) ? '' : $this->createRoute($overrideKeys);
+        $langShortCode = '';
```

Comment:
This is not needed is the variables should retrieve a value in the later if/else.